modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -16,7 +21,7 @@ def perform
         'status = ? AND created_at BETWEEN ? AND ? AND cid <> ?',
         'errored', @search_from, @search_to, '0oagdm49ygCSJTp8X297'
       ).pluck(:id).uniq
-      @va_gov_errored_claims = get_unique_errors
+      @va_gov_errored_claims = get_filtered_unique_errors # get_unique_errors
```

Comment:
Is this comment leftover?

Reply:
Yep it is cleaning it out

Reply:
[e0471a6](https://github.com/department-of-veterans-affairs/vets-api/pull/19273/commits/e0471a63dd2835b59ffec66c5f12b6e3aef11f34)